samples/matter/lock/README.rst

```diff
@@ -111,7 +109,7 @@ SEGGER J-Link USB port:
     Used for getting logs from the device or for communicating with it through the command-line interface.
 
 NFC port with antenna attached:
-    Optionally used for obtaining the commissioning information from the Matter device to start the :ref:`commissioning procedure <matter_lock_sample_remote_control>`.
+    Optionally used for obtaining the commissioning information from the Matter accessory device to start the :ref:`commissioning procedure <matter_lock_sample_remote_control>`.
```

Comment:
This change (and a few below) hasn't been made for the light bulb sample.

Reply:
Not sure which edits below you are referring to. Most of them are included in the light bulb sample with the `.. include::` command.